### ISSUE TITLE
Handle case when a subphrase subtends > 180deg

### ISSUE THREAD

**Issue by rossjrw** [rossjrw/gallifreyo#38]
When a subphrase occupies more than half of the circle, its radius is determined by the _smaller_ of the two angles, resulting in lots of lots space.

I see two solutions:

Quick and hacky: The maximum angle a phrase can subtend is 180deg; the remaining words fill the space. Problem: will be much harder to calculate absolute angle sizes.

Long-term solution: The centre of the circle moves away from the larger letter to more effectively move the space. I do not if this is even possible with the current design of the system, and if it is, I have absolutely no idea how to begin implementing it.

**Comment by rossjrw** [comment]
Thought: it's only the largest circle that needs to be changed, the rest can stay as they are. Instead of the centre of the circle moving, the dominant word can just artificially take up more space and overlap the midpoint. Could possibly be easier after #41.

**Comment by rossjrw** [comment]
Closing per #51. This is out of scope for the radial algorithm, and this is what the organic algorithm is intended to address.
